package com.java.learning.javase;

public class ExceptionSE {
    public static void main(String[] args) {
        // try wrapping the exception in a RuntimeException and log in
        
    }
    
}
